linera-rpc/src/grpc/conversions.rs

```diff
@@ -548,6 +577,89 @@ impl From<api::Blob> for Blob {
     }
 }
 
+impl TryFrom<api::CertificateValue> for CertificateValue {
```

Comment:
We should probably represent certificate values as BCS-serialized bytes; that will remove the redundant serialization when we compute the hash later.